miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleService.java

```diff
@@ -77,6 +79,8 @@
 
   private static final String ERR_MISSING_PARENT_ID = "Detailed sample is missing parent identifier";
 
+  private static final Pattern COMMA = Pattern.compile(",");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "COMMA" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)